.github/workflows/deploy_to_s3_prod.yml

```diff
@@ -36,15 +36,18 @@ jobs:
         NODE_ENV: Production
         VUE_APP_AF_API_KEY: ${{ secrets.VUE_APP_AF_API_KEY }}
         VUE_APP_BASE_URL_PLIO: ${{ secrets.VUE_APP_BASE_URL_PLIO }}
+        VUE_APP_AWS_ACCESS_KEY_ID: ${{ secrets.VUE_APP_AWS_SQS_ACCESS_KEY }}
+        VUE_APP_AWS_SECRET_KEY: ${{ secrets.VUE_APP_AWS_SQS_SECRET_ACCESS_KEY }}
+        VUE_APP_BASE_URL_MEET: ${{ secrets.VUE_APP_BASE_URL_MEET }}
       run: npm run build
 
     - name: Configure AWS Credentials
       uses: aws-actions/configure-aws-credentials@v1
       with:
-        aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-        aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+        aws-access-key-id: ${{ secrets.AWS_S3_ACCESS_KEY_ID }}
```

Comment:
I'm a bit confused. Are there three things? `AWS_KEY`, `AWS_SQS_KEY`, and `AWS_S3_KEY`?

Reply:
There are only 2 keys, one for SQS and one for S3

Reply:
There is a variable `VUE_APP_AWS_SECRET_KEY` in `.env.example`. That needs to be removed/renamed, then?